### ISSUE TITLE
image uploading doesn't appear to work

### ISSUE THREAD

**Issue by asg0451** [MohamedBassem/hoarder-app#12]
when i paste or drag an image into the "save" box, nothing happens. no console errors. chrome on mac

EDIT: i see it works from mac screenshot thing, but not from copy/paste or dragging from chrome. how is this feature intended to be used?

**Comment by asg0451** [comment]
also, i see this warning in the logs. is this ok?

```
 hoarder  ⨯ Error: 'sharp' is required to be installed in standalone mode for the image optimization to function correctly. Read more at: https://nextjs.org/docs/messages/sharp-missing-in-production      
```

**Comment by MohamedBassem** [comment]
This is how it's supposed to look like:

https://github.com/MohamedBassem/hoarder-app/assets/2418637/5dbc66f3-e62e-4c3b-be4a-024356e905bb

You get a visual indication that you're dragging stuff and when you drop them, they'll get uploaded. I've not implemented copy pasting though, I'll check if it's possible to implement.

I think beside drag and dropping, I can also add a button in the editor card that opens the file picker for a more reliable way to upload stuff.

**Comment by asg0451** [comment]
Ah I see. That sounds great, thanks!

This is a really cool project, seems well architected and I'd love to help out. But I'm a backend dev and idk ts 🤷 😂

**Comment by MohamedBassem** [comment]
@asg0451 Thanks!

Btw, I'm also a infrastructure engineer (and has been for the past 7 years). Had to learn a lot of stuff for this project :)

**Comment by asg0451** [comment]
that's really impressive, it looks great! the last time i made a frontend it looked... let's say retro ;)

**Comment by MohamedBassem** [comment]
The `sharp` error got fixed in 48169a0.
